core/src/main/java/com/linecorp/armeria/server/HttpServerPipelineConfigurator.java

```diff
@@ -242,6 +247,11 @@ private Http2Settings http2Settings() {
         return settings;
     }
 
+    void updateConfig(ServerConfig config) {
+        final ServerConfig cfg = requireNonNull(config, "config");
+        configHolder.replaceAndNotify(cfg);
```

Comment:
No need to introduce a local variable
```suggestion
        requireNonNull(config, "config");
        configHolder.replaceAndNotify(config);
```

Reply:
Done.